makeflow/src/makeflow_amazon_batch_setup

```diff
@@ -0,0 +1,179 @@
+#!/bin/sh
+#
+#Copyright (C) 2018- The University of Notre Dame
+#This software is distributed under the GNU General Public License.
+#See the file COPYING for details.
+#
+
+
+load_fieldINI()
+{
+	grep $1 $2 | cut -d "=" -f 2 | tr -d ' '
+}
+
+load_fieldJSON()
+{
+	grep $1 $2 | cut -d ":" -f 2 | tr -d ' ' | tr -d ','
+}
+
+if [ -z "$1" ]; then
+	echo "Empty number of CPUs requested."
+	echo "Correct Ussage: ./makeflow_amazon_batch_setup [desired_num_cpus] [min_number_cpus] [max_number_cpus] (config-output)"
+	exit 32
+fi
+
+if [ -z "$2" ]; then
+	echo "Empty number of min CPUs."
+	echo "Correct Ussage: ./makeflow_amazon_batch_setup [desired_num_cpus] [min_number_cpus] [max_number_cpus] (config-output)"
+	exit 33
+fi
+
+if [ -z "$3" ]; then
+	echo "Empty number of max CPUs."
+	echo "Correct Ussage: ./makeflow_amazon_batch_setup [desired_num_cpus] [min_number_cpus] [max_number_cpus] (config-output)"
+	exit 34
+fi
+
+cpus=$1
+min_cpus=$2
+max_cpus=$3
+time=$(date +%s)
+
+if [ $cpus -lt $min_cpus ]; then
```

Comment:
Same as before, no need to specialize the error code to the particular error.

Reply:
addressed with previous comment